corehq/ex-submodules/couchexport/templates/couchexport/html_export.html

```diff
@@ -8,8 +8,17 @@
 {% endif %}
 
 {% if section == "table_begin" %}
-    <h2>{{ name }}</h2>
-    <table>
+    <div>
+      <table>
+        <tr>
+          <td>
+      <h2>{{ name }}</h2>
```

Comment:
Sometimes I try to keep the diff as small as possible, and sometimes my OCD takes over and reformats huge chunks. One day I'll be able to find a balance. :balance_scale: